pkg/api/v1/types.go

```diff
@@ -1816,6 +1816,7 @@ type ServiceSpec struct {
 	// API for compatibility until at least 8/20/2016.  It will be removed from
 	// any new API revisions.  If both deprecatedPublicIPs *and* externalIPs are
 	// set, deprecatedPublicIPs is used.
+	// +genconversion=false
```

Comment:
I think we've been adding these tags in a separate block so they don't show up in docs? e.g.:

```
// +genconversion=false

// actual comment for docs
MyVariable string
```